RadixWallet/Features/SettingsFeature/DebugSettings/Coordinator/DebugSettingsCoordinator+View.swift

```diff
@@ -82,65 +75,79 @@ extension DebugSettingsCoordinator.View {
 
 // MARK: - Extensions
 
+private extension StoreOf<DebugSettingsCoordinator> {
+	var destination: PresentationStoreOf<DebugSettingsCoordinator.Destination> {
+		scope(state: \.$destination) { .child(.destination($0)) }
+	}
+}
+
+@MainActor
 private extension View {
-	@MainActor
-	func debugUserDefaultsContents(
-		with destinationStore: PresentationStoreOf<DebugSettingsCoordinator.Destinations>
+	func destinations(
+		with store: PresentationStoreOf<DebugSettingsCoordinator.Destination>
+	) -> some View {
+		factorSources(with: store)
+			.debugUserDefaultsContents(with: store)
+		#if DEBUG
```

Comment:
hmm not sure why I did it like this... bug this WHOLE feature should probabyl be put behind `#if DEBUG`, i.e. the whole `DebugSettingsCoordinator+`{Reducer,View} files.